AWSClientRuntime/Sources/Protocols/Ec2Query/Ec2Response.swift

```diff
@@ -12,5 +12,16 @@ public struct Ec2Response: Decodable {
     enum CodingKeys: String, CodingKey {
         case errors = "Errors"
         case requestId = "RequestId"
+        case requestID = "RequestID"
+    }
+    
+    public init(from decoder: Decoder) throws {
```

Comment:
Same question: Was this type using an auto-generated `init(from:)` before?

Reply:
yep!